client/src/components/Menu/MenuItem.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import withStyles from 'isomorphic-style-loader/withStyles';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import Link from '../Link/Link';
```

Comment:
Hmmm, you should import from simply `'../Link', but if this works, you can leave it as is